src/05-derivatives/05-functional-derivatives.md

```diff
@@ -0,0 +1,390 @@
+# Functional derivatives
+
+## Functional derivatives maps
+
+A derivative map is a measure derived from a functional series, mapped onto spatial
+locations as defined by voxels in a volume or vertices on a surface.
```

Comment:
Are these the only ways to define a location? If we have an electrode, for example, its localization can't be Cz, must be its coordinates, right? Are coordinates also included in this definition?